data/plays.py

```diff
@@ -0,0 +1,36 @@
+PLAY_RESULT_UPDATES = ["single", "double", "triple", "walk", "intent_walk", "home_run", "strikeout", "strikeout_looking"]
```

Comment:
There are other play results than just these - not that we'd want to show them necessarily, but code which uses this file should keep in mind that indexing into this or the dictionary can fail

Reply:
Agreed. I think the issue might be the naming of the list here. These are meant to be specific types of play results that will trigger the display change. Would naming this list appropriately address the concern, or do we want to move the logic to the method, only sending the sorts of play results that we want to trigger?

Reply:
Would prefer these to be constants, they are referred all throughout the code as magic strings. If we're going to index into a dict this should be mapped here:

```python
SINGLE   = "single"
DOUBLE   = "double"
TRIPLE   = "triple"
HOME_RUN = "home_run"

WALK             = "walk"
INTENTIONAL_WALK = "intent_walk"

STRIKEOUT         = "strikeout"
STRIKEOUT_LOOKING = "strikeout_looking"

HITS = [
  SINGLE,
  DOUBLE,
  TRIPLE
]

WALKS = [
  WALK,
  INTENTIONAL_WALK
]

STRIKEOUTS = [
  STRIKEOUT,
  STRIKEOUT_LOOKING
]
```

Reply:
Got it! It's done now.